modules/services/cloud-logs/main.tf

```diff
@@ -0,0 +1,63 @@
+#-----------------------------------------------------------------------------------------------------------------------
+# The only resource needed to make cloudingestion start to fetch data from the CloudTrail associated s3 bucket is a
+# properly set AWS IAM Role. Sysdig's trusted identity act as the Principal in the assume role Policy, namely the role
+# that CloudIngestion will use to assume the Client's role. At that point, given the permission set granted to the newly
+# created Role in the Client's account, CloudIngestion will be able to perform all the required actions in order to
+# list and fetch the log files that are automatically published in the target s3 bucket.
+#
+# Note: this setup assumes that the Customer has already properly set up an AWS CloudTrail Trail and the associated bucket.
+# It is responsibility of Sysdig's Secure UI to provide the necessary information to make the Customer perform the
+# necessary operations before applying the Terraform module.
+#-----------------------------------------------------------------------------------------------------------------------
```

Comment:
Please note this repo is public, and as such should not include internal implementation details, even in comments

Reply:
Got it, I'll generalize it a bit!